client/src/components/grants/Show.tsx

```diff
@@ -27,20 +27,23 @@ function Project() {
   }, shallowEqual);
 
   useEffect(() => {
-    if (props.ipfsInitialized && params.id) {
+    // called twice
```

Comment:
Maybe we can consolidate this and https://github.com/gitcoinco/grants-hub/pull/72/files#diff-98c909ece664b6cfe734fbadcfe5749ebdc02e5d20b98e02526bb09d98370900R80 into a component and use it while IPFS is loading? It could remove some of the conditions in the components. Or even a toast? wdyt?

Reply:
yeah how can we do it? cause we still need that to be called twice

Reply:
Yeah we could keep the call in the component but create a new component that shows the status, for this kind of stuff

`
  if (props.currentGrant === undefined && props.ipfsInitializationError) {
    return <>Error initializing IPFS. Reload the page and try again.</>;
  }

  if (props.currentGrant === undefined && !props.ipfsInitialized) {
    return <>Initializing ipfs...</>;
  }

  if (props.currentGrant === undefined && props.loading && props.currentGrant) {
    return <>Loading grant data from IPFS... </>;
  }
`